test/fs/proc.c

```diff
@@ -95,13 +95,16 @@ int main()
   describe("proc_read()");
   uptime = vfs_namei("/uptime");
   uint64_t bytes_read = vfs_read(uptime, buf, BUF_SIZE, 0);
+  buf[bytes_read] = '\0';
```

Comment:
Why would this need changes? This is the procfs and it should definitely return 0-terminated strings.

Reply:
Fixed